api-wrapper/requirements.txt

```
@@ -0,0 +1,4 @@
+requests
+dotenv
```

I believe you're looking for https://pypi.org/project/python-dotenv/
```suggestion
python-dotenv
```